### DISCUSSION TITLE
Download Error

### DISCUSSION THREAD

**Discussion by zafarnamah** [aclap-dev/video-downloadhelper#494] [Help and Q&A]
### Your question

I have bought a license but did not get a confirmation email. I did get a key after payment on the webpage, which was recognized by Google Chrome extension.  I was able to download one video but the second one on the same page gives this error:

SideDownload error:
[mp2 @ 0x1348fde90] Header missing
    Last message repeated 112 times
[mp4 @ 0x108d04ef0] sample rate not set
[out#0/mp4 @ 0x108d04e30] Could not write header (incorrect codec parameters ?): Invalid argument
Error opening output file /Users/harpreetsingh/Downloads/Course Resources.mp4.
Error opening output files: Invalid argument

### Addon & CoApp version and details

SideDownload error:
[mp2 @ 0x1348fde90] Header missing
    Last message repeated 112 times
[mp4 @ 0x108d04ef0] sample rate not set
[out#0/mp4 @ 0x108d04e30] Could not write header (incorrect codec parameters ?): Invalid argument
Error opening output file /Users/harpreetsingh/Downloads/Course Resources.mp4.
Error opening output files: Invalid argument

**Comment by paulrouget** [comment]
Can you please share the URL of the failing video, and also try the Beta version: https://chromewebstore.google.com/detail/video-downloadhelper-beta/pfoiagbblcbmognbkekfpodpidedkmcc

**Reply by zafarnamah** [reply]
This is the URL but it's protected by a login: https://vancouverinstitute.net/course-resources/%E0%A8%9D%E0%A8%A8%E0%A8%BE%E0%A8%82-%E0%A8%A6%E0%A9%80-%E0%A8%B0%E0%A8%BE%E0%A8%A4-%E0%A8%A6%E0%A9%80%E0%A8%B0%E0%A8%98-%E0%A8%85%E0%A8%A7%E0%A8%BF%E0%A8%90%E0%A8%A8

The Beta version works great but wants the premium subscription after the first download. I paste my license key but it does not work.  The regular version provides the first download and then gives the error I had provided.
